appengine/genetics.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="google" value="notranslate">
+  <meta name="viewport" content="target-densitydpi=device-dpi, width=device-width, initial-scale=1.0, user-scalable=no">
+  <title>Blockly Games : Genetics Lab</title>
+  <link rel="stylesheet" href="common/common.css">
+  <link rel="stylesheet" href="genetics/style.css">
+  <script src="third-party/ace/ace.js"></script>
+  <script src="third-party/ace/ext-language_tools.js"></script>
+  <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+  <script src="common/boot.js"></script>
+  <script src="/common/storage.js"></script>
```

Comment:
Slash is needed for storage.  This file must 404 when not run on App Engine
since storage is not available.

Send (awkwardly) from an Android phone.

On 27 Jul 2016 11:33 a.m., "Monica Kozbial" <notifications@github.com>
wrote:

> In appengine/genetics.html
> <https://github.com/google/blockly-games/pull/38#discussion_r72496597>:
>
> > @@ -0,0 +1,18 @@
> > +<!DOCTYPE html>
> > +<html>
> > +<head>
> > +  <meta charset="utf-8">
> > +  <meta name="google" value="notranslate">
> > +  <meta name="viewport" content="target-densitydpi=device-dpi, width=device-width, initial-scale=1.0, user-scalable=no">
> > +  <title>Blockly Games : Genetics Lab</title>
> > +  <link rel="stylesheet" href="common/common.css">
> > +  <link rel="stylesheet" href="genetics/style.css">
> > +  <script src="third-party/ace/ace.js"></script>
> > +  <script src="third-party/ace/ext-language_tools.js"></script>
> > +  <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
> > +  <script src="common/boot.js"></script>
> > +  <script src="/common/storage.js"></script>
>
> Doesn't look like the slash is necessary. Removed slash and did same
> change in pond-tutor.html
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/blockly-games/pull/38/files/2f2e526c42b25ce4d74436de259a615e43f54b79#r72496597>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAPScMNeEbfV40DmUPdtj5z_CbFYd0hBks5qZ6RrgaJpZM4JVs8W>
> .
>